pkgs/development/embedded/elf2uf2-rs/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, rustPlatform, fetchFromGitHub, pkg-config, udev }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "elf2uf2-rs";
+  version = "91ae98873ed01971ab1543b98266a5ad2ec09210";
```

Comment:
This is tooooo long for `version`. You may use `unstable-2021-12-12` or any version scheme proposed in [RFC 0107](https://github.com/NixOS/rfcs/blob/083116c0be0315977d7852b02b3f54c1fb766e62/rfcs/0107-version-normalization.md).

Reply:
Fixed